data/ME/incentives.json

```diff
@@ -358,14 +358,15 @@
     ],
     "program": "me_efficiencyMaine_efficiencyMaineResidentialHeatPumpIncentives",
     "amount": {
-      "type": "dollar_amount",
-      "number": 2000
+      "type": "percent",
+      "number": 0.85,
+      "maximum": 12900
     },
     "owner_status": [
       "homeowner"
     ],
     "short_description": {
-      "en": "$2,000 rebate for ducted heat pump, with option to be distributed in monthly payments. Available for low income residents."
+      "en": "Up to 85% of the cost of ducted heat pump systems for manufactured (mobile) homes. Available for low-income households. $2,000 co-pay."
```

Comment:
Urgh, I wonder if co-pays are something we have to start representing... I think we're OK without it for now, but that may change if we see more incentives like this.

Reply:
yeah I agree. I'll make a ticket to track the concept